src/runtime_src/core/tools/xbutil2/SubCmdReset.cpp

```diff
@@ -19,6 +19,7 @@
 #include "xrt.h"
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.